app/src/main/AndroidManifest.xml

```diff
@@ -100,7 +100,10 @@
                 android:value=".OrbotMainActivity" />
         </activity>
 
-        <activity android:name=".ui.onboarding.OnboardingActivity" />
+        <activity
+            android:name=".ui.onboarding.OnboardingActivity"
+            android:configChanges="orientation|screenSize|screenLayout|keyboardHidden" />
```

Comment:
I guess we don't need the keyboard, layout, screensize flags? we just want to save the state on orientation change.

Reply:
I think screenSize and screenLayout are needed, but not keyboardHidden.

https://developer.android.com/guide/topics/resources/runtime-changes